desktop/updater/index.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+/**
+ * External Dependencies
+ */
+const electron = require( 'electron' );
+const app = electron.app;
+
+/**
+ * Internal dependencies
+ */
+const platform = require( '../platform' );
+const config = require( '../config' );
+const AutoUpdater = require( './auto-updater' );
+const ManualUpdater = require( './manual-updater' );
+// const settings = require( 'lib/settings' );
+
+let updater = false;
+
+function urlBuilder( version ) {
+	let platformString = 'osx';
+
+	if ( platform.isWindows() ) {
+		platformString = 'windows';
+	} else if ( platform.isLinux() ) {
+		platformString = 'linux';
+	}
+
+	if ( config.forceUpdate ) {
+		version = '0.0.1';
+	}
+
+	return config.updater.url + platformString + '/version?compare=' + version;
```

Comment:
it would be nice to see us using a URL path builder here just to prevent issues with the slashes and query parts. for example, if we mistype coming in `config.updater.url`

```js
import joinUrl from 'url-join';

return joinUrl( config.updater.url, platformString, 'version?compare=' + version );
```

or a step further…

```js
import url from 'url';
import joinUrl from 'url-join';

return url.format( {
	...url.parse( joinUrl( config.updater.url, platformString, 'version' ) ),
	query: { compare: version }
} );
```

Unfortunately it doesn't look like `url` handles joining path pieces in a way that prevents double slashes.